eclipse/ide-target-platform/category.xml

```diff
@@ -37,5 +37,5 @@
    <bundle id="com.fasterxml.jackson.core.jackson-core" version="0.0.0"/>
    <bundle id="ch.qos.logback.slf4j" version="0.0.0"/>
    <bundle id="org.slf4j.log4j" version="0.0.0"/>
-   <bundle id="org.yaml.snakeyaml" version="1.17"/>
+   <bundle id="org.yaml.snakeyaml" version="1.17.0"/>
```

Comment:
Should just use 0.0.0 so as to match the version defined in the target platform.  Version 0.0.0 will be replaced by Tycho to that matched at build time.  We only need to specify an explicit version if there may be multiple bundles resolved.